server/src/main/java/org/elasticsearch/action/admin/cluster/shards/ClusterSearchShardsRequest.java

```diff
@@ -20,6 +20,10 @@
 import java.io.IOException;
 import java.util.Objects;
 
+/**
+ * Use {@link org.elasticsearch.action.search.SearchShardsRequest} instead
+ */
+@Deprecated
```

Comment:
let's do this as a follow-up? Feels like there is more to deprecate than just the request class, probably the API overall.

Reply:
++, I reverted this in https://github.com/elastic/elasticsearch/pull/94534/commits/234565f630cee42058a56fb986b5e1688e707037